src/modules/products/screens/detail-products/details-screen.tsx

```diff
@@ -0,0 +1,105 @@
+import { useNavigation } from '@react-navigation/native';
+import { styled } from 'nativewind';
+import React, { useState } from 'react';
+import { Image as RNImage, TouchableOpacity as RNTouchableOpacity } from 'react-native';
+
+import { common } from '../../../../translations/en.json';
+import DropdownComponent from '../../../../ui/core/select-menu';
+import { Text } from '../../../../ui/core/text';
+import { SafeAreaView, ScrollView, View } from '../../../../ui/core/view';
+const Image = styled(RNImage);
+const TouchableOpacity = styled(RNTouchableOpacity);
+
+const item = {
+  state: 'Restored',
+  name: 'Baumler Chair',
+  manufactur: 'Woodstock Co.',
+  price: 30,
+  description1:
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam eu turpis molestie, dictum est a, mattis tellus. Sed dignissim, metus nec fringilla accumsan, risus sem sollicitudin lacus, ut interdum tellus elit sed risus.\n',
+  description2:
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit etiam eu turpis molestie, dictum est a, mattis tellus.',
+};
+
+export const DetailsScreen = () => {
+  const availableItems = 12;
```

Comment:
Remember to define component-independent values outside of it

Reply:
Done.